### DISCUSSION TITLE
Server error when adding code snippet

### DISCUSSION THREAD

**Discussion by leoplusx** [paircodedev/sandbox#5] [Bug Report] [answered]
After clicking "Add code snippet", I get to the screen where you can paste code. I select "HTML" as the language and paste my code. An error message appears, saying something like "Error 500. Click here to go back to the homepage". It not only doesn't save my code snippet, but also takes me out of the process for posting my bounty. (Luckily, hitting the "back" button in the browser does take me back into the process, and my changes prior to adding a code snippet were saved.)

**Comment by Necmttn** [comment, marked as answer]
Hey, @leobaumgardt thank you for reporting! I just pushed a new release that resolves it.
🎉

**Reply by leoplusx** [reply]
Lovely! 👍
